.github/workflows/megalinter.yaml

```
@@ -30,6 +30,7 @@ env:
   # If APPLY_FIXES is used, defines if the fixes are directly committed (commit)
   # or posted in a PR (pull_request)
   APPLY_FIXES_MODE: commit
+  APPLY_FIXES_IF_COMMIT: ${{ env.APPLY_FIXES == 'all' && env.APPLY_FIXES_MODE == 'commit' }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)